actionview/test/template/url_helper_test.rb

```diff
@@ -50,6 +50,8 @@ class UrlHelperTest < ActiveSupport::TestCase
   include RenderERBUtils
 
   setup :_prepare_context
+  setup    { ActionView::Helpers::UrlHelper.button_to_generates_button_tag = true }
```

Comment:
Let's store the old value in an instance variable and reset the value of the config to it in a teardown. This way if we change the default to `true` we set back to `true` instead of to `false`.